charts/gardener-dashboard/values.yaml

```diff
@@ -41,6 +41,8 @@ ingress:
     nginx.ingress.kubernetes.io/use-port-in-redirects: "true"
     kubernetes.io/ingress.class: nginx
 tls:
+  enabled: true
+  # secretName: gardener-dashboard-tls
```

Comment:
```suggestion
```

I think it is better to keep the value `tlsSecretName` on top level and do not introduce the enabled flag as part of the tls section. The tls section should only contain information for the gardener-dashboard tls-secret and not for the  ingress.

```
ingress:
  ...
tlsSecretName: gardener-dashboard-tls
tls:
  ...
```